source/standalone/workflows/rl_games/play.py

```diff
@@ -140,6 +160,10 @@ def main():
                 if agent.is_rnn and agent.states is not None:
                     for s in agent.states:
                         s[:, dones, :] = 0.0
+        if args_cli.video:
+            timestep += 1
+            if timestep == args_cli.video_length:
+                break
```

Comment:
Can you please add some comments above on why this is needed? Is it just to record one video and then exit the play loop?